cli/src/commands/utils.ts

```diff
@@ -85,3 +114,40 @@ export function validateAddress(address: string) {
     process.exit(1)
   }
 }
+
+interface WalletConfig {
+  wallet: string
+  password: string
+}
+
+interface Config {
+  wallet_config: WalletConfig
```

Comment:
I don't really like dividing this into 2 parts, especially as I fell like it doesn't bring much value. Either we put everything related to wallet in it (including the `secret_key`), either we keep everything at the "root" of the file like:
```suggestion
interface Config {
  wallet_path: string
  wallet_password: string
```

IMO, it's cleaner like that as it is easier to right in the file and we don't have much options for now. If one day we end up with dozens of options, we will clearly divide the config file into smaller sections